js/renderingPicture.js

```diff
@@ -0,0 +1,37 @@
+import {createPhotos} from './mockPhotos.js';
+
+const dataCreatePicture = createPhotos();
+const SAMPLE = document.querySelector('#picture').content.cloneNode(true);
+const PICTURES = document.querySelector('.pictures');
+const fragment = document.createDocumentFragment();
+
+const createPictures = (data, sample) =>{
+  const sampleClone = sample.cloneNode(true);
```

Comment:
Мы просто можем взять переменную samplePicture "снаружи" этой функции
```suggestion
  const sampleClone = samplePicture.cloneNode(true);
```

Reply:
+